tests/test_script.py

```diff
@@ -53,3 +54,16 @@ def test_min_confidence():
     assert call_vulture([
         'vulture/core.py', '--exclude', 'whitelists',
         '--min-confidence', '100']) == 0
+
+
+def test_non_utf8_encoding():
+    with tempfile.NamedTemporaryFile(mode='wb', dir=REPO, suffix='.py') as f:
+        code = """\
+def foo():
+    pass
+
+foo()
+"""
+        f.write(code.encode("utf_16_be"))
+        f.close()
+        assert call_vulture([f.name]) == 1
```

Comment:
Can we dedent the last line and remove the call `f.close()` (which should happen automatically after exiting the with-block)?

Reply:
As soon as we are out of the `with` block, `tempfile` would delete the file created. This would cause a false negative, as exit code `1` would be because of file not found error.

Reply:
OK, thanks. I think it would be best to avoid such possible confusion by using the API, not the CLI to test non-UTF8 files.

Reply:
Yes, agreed!

Reply:
I agree.